build.gradle.kts

```diff
@@ -14,28 +14,13 @@ configurations {
   }
 }
 
-// spring boot configuration specifies the version of selenium so need to override for serenity to work properly
-ext["selenium.version"] = "4.1.1"
-
-// SDI-261: Pin versions to prevent issue with useLatestVersions
-
-// Problem with hibernate-core 5.6.7.Final which was causing issue with 'startingWith'
-// https://github.com/spring-projects/spring-data-jpa/issues/2472
-// https://hibernate.atlassian.net/browse/HHH-15142
-// Temporarily revert to 5.6.5.Final until fixed
-val hibernateCoreVersion by extra("5.6.5.Final")
-
 // Temporarily kept at 4.3 due to bug in 4.4 parser
 val jsqlParserVersion by extra("4.3")
 
 // Temporarily keep at 2.5.1 until can switch to h2 instead (tests break anyway with 2.6.1)
 val hsqldbVersion by extra("2.5.1")
 
-// groovy errors with latest version
-val restAssuredVersion by extra("4.5.1")
-
-// Temporarily keep at 3.2.2 as seems to bring in groovy incompatibilites by upgrading to 3.2.4
-val serenityVersion by extra("3.2.2")
+ext["rest-assured.version"] = "5.1.1"
```

Comment:
turns out this is pinned by spring boot to 4.5.1, so need to override in order to avoid dependency conflict